cmd/create/accountroles/cmd.go

```diff
@@ -409,6 +405,11 @@ func run(cmd *cobra.Command, argv []string) {
 		isHostedCPValueSet = true
 	}
 
+	if !createHostedCP {
+		rosa.HostedClusterOnlyFlag(r, cmd, route53RoleArnFlag)
+		rosa.HostedClusterOnlyFlag(r, cmd, vpcEndpointRoleArnFlag)
+	}
+
```

Comment:
Moved this here so it will only check if not also creating hostedCP roles (if creating both)